src/pykeen/datasets/cli.py

```diff
@@ -36,23 +41,28 @@ def summarize():
             click.secho(str(e), fg="red", bold=True)
 
 
-def _iter_datasets(regex_name_filter=None):
+def _iter_datasets(regex_name_filter=None, max_triples: Optional[int] = None) -> Iterable[Tuple[str, Type[Dataset]]]:
+    def _get_num_triples(pair: Tuple[str, Type[Dataset]]) -> int:
```

Comment:
remove this function from the inside of the bigger function

Reply:
c93c16c